flixel/input/gamepad/mappings/SwitchJoyconRightMapping.hx

```diff
@@ -77,19 +77,19 @@ class SwitchJoyconRightMapping extends FlxGamepadMapping
 	}
 
 	override function getInputLabel(id:FlxGamepadInputID)
-	{
+{
```

Comment:
? :smile:

Reply:
odd! I can fix. What is the token to prevent a CI check?